templates/test/ci/cluster-template-prow-ci-version-windows.yaml

```diff
@@ -422,14 +422,14 @@ apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: AzureMachineTemplate
 metadata:
   annotations:
-    runtime: dockershim
+    runtime: containerd
```

Comment:
This file can be deleted. It was only used by dockershim setup.  containerd uses the same prow-template as Linux

Reply:
done